docs/help/faq.md

```diff
@@ -30,16 +30,19 @@ App][feature-app-in-feature-app].
 ## Can we integrate an existing Angular 2.0 application into the Feature Hub?
 
 Yes, but the application must first be [converted to a Feature
-App][writing-a-feature-app]. The [`@feature-hub/react`][react-api] package could
-be used to integrate the application as a [DOM Feature App][dom-feature-app].
-However, if you don't want to integrate other React Feature Apps, you probably
-should build a Feature Hub Integrator without React.
-
-[own-feature-service-definitions]:
-  /docs/guides/writing-a-feature-app#ownfeatureservicedefinitions
+App][writing-a-feature-app]. The [`@feature-hub/dom`][dom-api] or the
+[`@feature-hub/react`][react-api] package can be used to integrate the
+application as a DOM Feature App. We have created a [demo][angular-feature-app]
```

Comment:
Do we want to write as `we` here or, do we just want to say that "There is a demo..."?

Reply:
Yes you're right

Reply:
done